tf_encrypted/session.py

```diff
@@ -13,10 +13,10 @@
 from .tensor.factory import AbstractTensor
 
 
-__TFE_STATS__ = bool(os.getenv('TFE_STATS', False))
-__TFE_TRACE__ = bool(os.getenv('TFE_TRACE', False))
-__TFE_DEBUG__ = bool(os.getenv('TFE_DEBUG', False))
-__TENSORBOARD_DIR__ = str(os.getenv('TFE_STATS_DIR', '/tmp/tensorboard'))
+__tfe_events__ = bool(os.getenv('TFE_EVENTS', ""))
```

Comment:
why `""` instead of `False`?

Reply:
Linting expects a string there, and tests were failing with `"False"`

Reply:
perfect